webapp/src/assets/styles/index.css

```diff
@@ -12,10 +12,11 @@
 }
 
 .leaflet-container {
-  height: 500px;
+  /* This is approximately equal to the leftover space after accounting for the header */
+  height: calc(100vh - 150px);
 }
 
-.scooterCluster {
+.mobility-map-cluster-icon {
```

Comment:
Please use consistent css class naming.

I would also suggest using scss for this and splitting it into separate files.